fake_quant cachemask: remove Python bindings

vkuzo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#51878 fake_quant cachemask: remove Python bindings**

Summary:

`fake_quantize_per_tensor_affine_cachemask` and
`fake_quantize_per_channel_affine_cachemask` are implementation
details of `fake_quantize_per_tensor_affine` and
`fake_quantize_per_channel_affine`, removing the
Python bindings for them since there is no need to
expose them.

Test Plan:

```
python test/test_quantization.py TestFakeQuantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D26314173](https://our.internmc.facebook.com/intern/diff/D26314173)

vkuzo committed:
fake_quant cachemask: remove Python bindings

Summary:

`fake_quantize_per_tensor_affine_cachemask` and
`fake_quantize_per_channel_affine_cachemask` are implementation
details of `fake_quantize_per_tensor_affine` and
`fake_quantize_per_channel_affine`, removing the
Python bindings for them since there is no need to
expose them.

Test Plan:

```
python test/test_quantization.py TestFakeQuantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

albanD reviewed (approved):
Awesome!
Thanks for taking the time to clean that up!

vkuzo committed:
Update on "fake_quant cachemask: remove Python bindings"


Summary:

`fake_quantize_per_tensor_affine_cachemask` and
`fake_quantize_per_channel_affine_cachemask` are implementation
details of `fake_quantize_per_tensor_affine` and
`fake_quantize_per_channel_affine`, removing the
Python bindings for them since there is no need to
expose them.

Test Plan:

```
python test/test_quantization.py TestFakeQuantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D26314173](https://our.internmc.facebook.com/intern/diff/D26314173)

[ghstack-poisoned]

vkuzo committed:
Update on "fake_quant cachemask: remove Python bindings"


Summary:

`fake_quantize_per_tensor_affine_cachemask` and
`fake_quantize_per_channel_affine_cachemask` are implementation
details of `fake_quantize_per_tensor_affine` and
`fake_quantize_per_channel_affine`, removing the
Python bindings for them since there is no need to
expose them.

Test Plan:

```
python test/test_quantization.py TestFakeQuantize
```

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D26314173](https://our.internmc.facebook.com/intern/diff/D26314173)

[ghstack-poisoned]
